scripts/upgrade

```diff
@@ -65,47 +26,11 @@ ynh_script_progression --message="Ensuring downward compatibility..."
 
 # If language_key doesn't exist, create it
 if [ -z "$language_key" ]; then
-	language_key=$(ynh_app_setting_get --app=$app --key=language)
+        language_key=$(ynh_app_setting_get --app=$app --key=language)
 	ynh_app_setting_set --app=$app --key=language_key --value=$language_key
 	ynh_app_setting_delete --app=$app --key=language
```

Comment:
Hmmm from what we see in the install script, it's converted from eg `fr` to `french` or `english` ... or is the old setting already transformed ?

Reply:
Yes indeed, on master the language -> language key is already done.